fxmanifest.lua

```diff
@@ -9,6 +9,7 @@ ui_page 'html/index.html'
 shared_scripts {
     '@qb-core/shared/locale.lua',
     'locales/en.lua', -- Change to the language you want
```

Comment:
Please remove the comment, there is no need for the user to change this anymore.